### PR TITLE
Add running interaction tests as part of CI

### PR DESCRIPTION
**Pull request by marcelwgn**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Continued work on #7236 

**Details of Changes**
Add details of changes here.
- Adding steps to run interaction tests as part of the CI

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by marcelwgn** [comment]
@yair100 For the interaction tests to properly work in Azure Devops, I think we need to install the WinAppDriver extension: https://marketplace.visualstudio.com/items?itemName=WinAppDriver.winappdriver-pipelines-task

**Comment by yaichenbaum** [comment]
@chingucoding just installed 👍

**Comment by marcelwgn** [comment]
Awesome, thank you @yair100 ! (I'll continue tomorrow with this PR)

**Comment by chingucoding** [comment]
azp /run

**Comment by chingucoding** [comment]
I think this PR is ready for review now. Tests ran fine with the release version of Files.

File: src/Files/Views/SettingsPages/Experimental.xaml [hunk 1]
```diff
@@ -67,7 +67,7 @@
                 <local:SettingsBlockControl.Icon>
                     <FontIcon Glyph="&#xE2B2;" />
                 </local:SettingsBlockControl.Icon>
-                <ToggleSwitch IsOn="{Binding ShowFolderSize, Mode=TwoWay}" Style="{StaticResource RightAlignedToggleSwitchStyle}" />
+                <ToggleSwitch IsOn="{Binding ShowFolderSize, Mode=TwoWay}" AutomationProperties.Name="{helpers:ResourceString Name=ShowFolderSize}" Style="{StaticResource RightAlignedToggleSwitchStyle}" />
```
Comment (yaichenbaum) [inline src/Files/Views/SettingsPages/Experimental.xaml:70]: Nice catch!

**Review by yair100 — APPROVED** [review]
Very nice work! LGTM

**Pull request merged**